core/domain/ecosystem/models/ecosystem-project-list-item-model.ts

```
@@ -0,0 +1,16 @@
+import { ProjectListItemV2 } from "../../project/models/project-list-item-model-v2";
+
+export type EcosystemProjectListItemResponse = components["schemas"]["ProjectShortResponseV2"];
+
+export interface EcosystemProjectListItemInterface extends EcosystemProjectListItemResponse {
+  categories: ProjectCategoryInterface[];
+}
+
+export class EcosystemProjectListItem extends ProjectListItemV2 implements EcosystemProjectListItemInterface {
+  constructor(props: EcosystemProjectListItemResponse) {
+    super(props);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)